test/app.test.js

```diff
@@ -1,10 +1,11 @@
-import { getNearestShops } from '../src/app';
+import {getNearestShops} from '../src/app';
+
 
 describe('App', () => {
-  it('should return an array when the input is valid', () => {
-    expect(Array.isArray(getNearestShops({
-      lat: 0,
-      lng: 0,
-    }))).toBe(true);
-  });
+    it('should return an array when the input is valid', async () => {
+        expect(Array.isArray(await getNearestShops({
+            lat: 0,
+            lng: 0,
+        }))).toBe(true)
+    });
```

Comment:
This hints that you have a coding style enforced using a linter and a formatter, I like that.